addons/l10n_ec/__init__.py

```diff
@@ -2,3 +2,4 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 from . import models
 from . import demo
+from . import tests
```

Comment:
I'm pretty sure this isn't necessary! :smile: Otherwise, all good!

Reply:
oupsie automatism